schema.sql

```diff
@@ -80,4 +80,11 @@ CREATE TABLE visits (
     vet_id INT REFERENCES vets(id),
     visit_date DATE,
     PRIMARY KEY (animal_id, vet_id, visit_date)
-);
\ No newline at end of file
+);
+
+--Add an email column to your owners table
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX audit_visits ON visits (animal_id, vet_id);
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly ensure the query output with indexes added, should have an index scan. With an index scan in the output, you can ascertain that indexes were indeed added. This will significantly increase your query performance.